core/error.rs

```diff
@@ -96,6 +96,33 @@ pub fn get_custom_error_class(error: &Error) -> Option<&'static str> {
   error.downcast_ref::<CustomError>().map(|e| e.class)
 }
 
+/// A wrapper around `anyhow::Error` that implements `std::error::Error`
+#[repr(transparent)]
+pub struct StdAnyError(pub Error);
```

Comment:
Annoyingly, `anyhow::Error` doesn't impl the `Error` trait. Not sure if we want to go down the error enum route (like `serde_v8::Error`) or continue using `anyhow::Error` as our primary error type.